docs/sw-cli.md

```diff
@@ -36,6 +36,20 @@ After the command has run you'll have two new files, a
 `sw-lib.vX.X.X.min.js` and `sw.js` file (unless you changed
 the file name).
 
+## Creating a Config File
+
+One of the questions asked by `sw-cli generate:sw` is whether you'd like to save
+your responses to a configuration file. Answering `Y` will save your responses
+to `sw-cli-config.json`. Subsequent runs of the cli will regenerate the service
+worker based on the config file. Not only does this make building faster, it
+opens the possibility of running subsequent calls in a build script.
+
+For example, if you're using gulp you can pass `sw-cli generate:sw` to
```

Comment:
I think both options have their place but that using sw-build would probably address the 90% case without them having to think about cli. Perhaps we steer them towards build and mention the cli workflow here as an alternative?